app/components/field.tsx

```diff
@@ -34,8 +34,28 @@ export function Field() {
     );
   }, [size]);
 
-  // todo: add skeleton loader
-  return !plots ? null : (
+  return !plots ? (
+    <div className="absolute bg-white h-full w-full left-0 right-0">
```

Comment:
may not need `left-0` and `right-0` if you have `h-full` and `w-full` (which can be `w-[100dvw]` `h-[100dvh]`.
also, do we need z-index here?